app/src/main/res/layout/fragment_profile.xml

```diff
@@ -8,182 +8,175 @@
 
     <ImageView
         android:id="@+id/imageView"
-        android:layout_width="376dp"
-        android:layout_height="113dp"
-        android:layout_marginTop="20dp"
+        android:layout_width="wrap_content"
+        android:layout_height="100dp"
```

Comment:
Parametrizzare valori come questo in dimens.xml

Reply:
Aggiornati